libcxx/include/__node_handle

```diff
@@ -179,6 +180,8 @@ struct __map_node_handle_specifics {
   using key_type    = __remove_const_t<typename _NodeType::__node_value_type::first_type>;
   using mapped_type = typename _NodeType::__node_value_type::second_type;
 
+  // https://www.open-std.org/jtc1/sc22/wg21/docs/papers/2025/p3372r3.html#node-handle-key
```

Comment:
Please add some description here. Just two links are really not helpful if you just look at the code. A single link would also be good enough IMO.

Reply:
Updated in 27e2117f1ad3

let me know if this looks good